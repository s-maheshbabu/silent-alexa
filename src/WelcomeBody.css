/** Scaffolding **/
#welcomeBody {
  grid-column: 1 / 3;
  grid-template-rows: auto;
  height: calc(100vh - 60px);
}

.panel-body {
  overflow: auto;
  text-align: center;
}
