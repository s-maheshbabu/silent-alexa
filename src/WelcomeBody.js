import React from "react";
import "./WelcomeBody.css";

export default function WelcomeBody(props) {
  return (
    <div id="welcomeBody">
      <div className="panel-body">
        <h1> WELCOME TO SILENT ALEXA </h1>
        <h1> Login to get started </h1>
      </div>
    </div>
  );
}
