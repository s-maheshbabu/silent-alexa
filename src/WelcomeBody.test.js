import React from "react";
import { shallow } from "enzyme";
import WelcomeBody from "./WelcomeBody";

it("renders without crashing", () => {
  const wrapper = shallow(<WelcomeBody />);
  expect(wrapper).toMatchSnapshot();

  wrapper.unmount();
});
