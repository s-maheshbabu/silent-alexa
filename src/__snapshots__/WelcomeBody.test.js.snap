// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  id="welcomeBody"
>
  <div
    className="panel-body"
  >
    <h1>
       WELCOME TO SILENT ALEXA 
    </h1>
    <h1>
       Login to get started 
    </h1>
  </div>
</div>
`;
